setup.py

```diff
@@ -41,23 +41,17 @@
     long_description=open('README.rst').read(),
     classifiers=[
         'Framework :: Django',
-        'Framework :: Django :: 1.10',
         'Framework :: Django :: 1.11',
-        'Framework :: Django :: 1.8',
-        'Framework :: Django :: 1.9',
         'Framework :: Django :: 2.0',
-        'Framework :: Django :: 2.1',
```

Comment:
Looks like this particular package still supports Django 1.8+, so I would leave all these in.